### ISSUE TITLE
📚 Documentation: Buttons do not have sufficient contrast using Appwrite Pink [a11y][TheA11y100]

### ISSUE THREAD

**Issue by GrahamTheDevRel** [appwrite/website#293]
## Issue
The Appwrite pink with white text does not have sufficient contrast (currently #FC356E to #ffffff is a contrast ratio of 3.6:1). 

!["Go to console" button screenshot](https://github.com/appwrite/website/assets/121672086/5cca3e5b-c985-4ee0-af7f-d6972b67962a)

## Why it matters
There are millions of people who have poor contrast perception, so it helps make text legible and the site easier to use.

## Suggested fix
Check with Appwrite design team, but one possible fix (if unable to use a darker background colour) would be to change the button text to be 19px and bold. This would then pass as "large text" under WCAG AA.

Another option would be black text on the Appwrite Pink (doesn't look great!).

## Relevant Success Criterion WCAG
[Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

## Additional Notes
@jasonetorres is interested in this one

[Found on stream on 2023-10-25](https://www.twitch.tv/videos/1959992199)

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by jasonetorres** [comment]
@TorstenDittmann Love to chat and get this assigned

**Comment by TGlide** [comment]
@jasonetorres we need to check with design team beforehand. Will come back to it

**Comment by CBID2** [comment]
Can I work on this @jasonetorres? I haven't gotten a chance to contribute to Appwrite yet and I really want to :)

**Comment by jasonetorres** [comment]
When @TGlide clears it for assignment you can take it @CBID2 but you owe me now muahaha

**Comment by CBID2** [comment]
> When @TGlide clears it for assignment you can take it @CBID2 but you owe me now muahaha

Sure @jasonetorres. Any contribution you make to [opensauced/pizza-verse](https://github.com/open-sauced/pizza-verse), I'll review it!.

**Comment by CBID2** [comment]
Hey @TGlide. It's been a month since this issue has been posted & no progress has been made. Can I work on it?

**Comment by jasonetorres** [comment]
Definitely take it @CBID2 i got a bit backlogged

**Comment by CBID2** [comment]
> Definitely take it @CBID2 i got a bit backlogged 

Thanks @jasonetorres! 😁

**Comment by TGlide** [comment]
@CBID2 we're still waiting for work by our design team to move on this issue :)

**Comment by CBID2** [comment]
Hey @TGlide! :) Any update from the design team yet?

**Comment by eldadfux** [comment]
Starting today this repo is archived. Please raise any issues with the new Appwrite website on the main platform repo: https://github.com/appwrite/appwrite
